packages/components/organisms/f-cookie-banner-static/CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v1.2.0
+------------------------------
+*July 26, 2022*
+
+### Updated
```

Comment:
NIT. Changed.

Reply:
Working in this component in another branch, so will update it then, just to save me having to run another build!